README.md

```diff
@@ -129,6 +129,9 @@ truffle(optimism_goerli)> share_instance.addApprovedBuild("0x0000000000000000000
 - `SHARE026` : `A non-zero licensing price was supplied but this contract does not support licensing.`
 - `SHARE027` : `Collection price per license must be >= the maximum license price of all items in the collection.`
 - `SHARE028` : `The tokenId and addresses liquid split arrays must be the same length.`
+- `SHARE029` : `Community splits initialization percentage cannot exceed 10000 basis points (100%).`
```

Comment:
We should make language consistent, eg. if the units are basis points, then the suffix of the variable should be basis points rather than percentage. That said, is there a reason we'd use basis points here?

Reply:
I believe basis points to be a more consistent standard. I was thinking that off hand two people might interpret percentage in different ways.
eg 2 people want to do 50% community splits, one person sends the contract .5 and the other 50. We can add verification logic that would be similar to the basis point verification but I think basis points will consistently be interpreted correctly.

Write
Preview
Attach files by dragging & dropping, selecting or pasting them.
Cancel
Update comment